.travis.yml

```diff
@@ -6,16 +6,9 @@ matrix:
   include:
   - os: linux
     compiler: gcc
-    env: CC_OQS=gcc-4.8 ENABLE_SIDH_IQC_REF=1
-    addons:
-      apt:
-        sources:
-        - ubuntu-toolchain-r-test
-        packages:
-        - gcc-4.8
-        - libgmp-dev
+    env: ENABLE_SIDH_IQC_REF=1
```

Comment:
Is there a reason you took out CC=gcc-4.8?  Is the idea to have this target have building with the default compiler?